relays/bin-substrate/src/cli/encode_message.rs

```diff
@@ -42,11 +43,16 @@ pub enum Message {
 pub type RawMessage = Vec<u8>;
 
 pub trait CliEncodeMessage: Chain {
-	/// Encode a send XCM call of the XCM pallet.
-	fn encode_send_xcm(
-		message: xcm::VersionedXcm<()>,
-		bridge_instance_index: u8,
+	/// Encode an `execute` XCM call of the XCM pallet.
+	fn encode_execute_xcm(
+		message: xcm::VersionedXcm<Self::Call>,
 	) -> anyhow::Result<EncodedOrDecodedCall<Self::Call>>;
+
+	/// Estimate value of `max_weight` argument for the `execute` XCM call of the XCM pallet.
+	fn estimate_execute_xcm_weight() -> Weight {
```

Comment:
We're now using `execute` instead of `send` on testnets - it is easier in configuration and the XCM program is similar to what we actually have on bridge hubs.